proxy/client.go

```diff
@@ -64,13 +65,17 @@ func (r *remoteClientCreator) NewABCIClient() (abcicli.Client, error) {
 
 func DefaultClientCreator(addr, transport, dbDir string) ClientCreator {
 	switch addr {
-	case "kvstore":
-		fallthrough
+	case "counter":
+		return NewLocalClientCreator(counter.NewCounterApplication(false))
+	case "counter_serial":
+		return NewLocalClientCreator(counter.NewCounterApplication(true))
 	case "dummy":
-		return NewLocalClientCreator(kvstore.NewKVStoreApplication())
-	case "persistent_kvstore":
 		fallthrough
+	case "kvstore":
+		return NewLocalClientCreator(kvstore.NewKVStoreApplication())
 	case "persistent_dummy":
+		fallthrough
+	case "persistent_kvstore":
 		return NewLocalClientCreator(kvstore.NewPersistentKVStoreApplication(dbDir))
 	case "nilapp":
```

Comment:
I don't like the name to be honest. Maybe rename to `empty` (or `noop`)

Reply:
Great idea - let's call it `noop`. And we should deprecate `dummy` and `nillapp` names for v0.28